hangman_gui.py

```diff
@@ -0,0 +1,266 @@
+import Tkinter
+import csv
+import random
+import sys
+
+# TO ADD:
+# HIGH SCHORE: can see high scores, writes correctly
+# NEW GAME: text changes so used can see new theme and such
+
+wordList = open("C:\Users\Steph\Documents\GitHub\Hangman\wordList.csv")
+csv_wordList = csv.reader(wordList)
```

Comment:
csv seems overkill for this, you could just do
theme:word1,word2,word3,word4 etc.
then
theme, words = line.split(":")
wordBox[theme] = words.split(",")